components/main_menu/main_menu.jsx

```diff
@@ -148,6 +148,7 @@ class MainMenu extends React.PureComponent {
                 mobile={this.props.mobile}
                 id={this.props.id}
                 ariaLabel={formatMessage({id: 'navbar_dropdown.menuAriaLabel', defaultMessage: 'main menu'})}
+                customStyles={{left: '-44px', marginLeft: 0}}
```

Comment:
It seems to me, like providing styles object is a preferred solution, but wouldn't it be better to give className prop to `Menu` object to keep styles in `.scss` files?

Reply:
I modified `sass/layout/_headers.scss` instead.